x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/kibana_tab_content.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC } from 'react';
+import React, { useEffect, useState } from 'react';
+import {
+  EuiAccordion,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiLink,
+  EuiNotificationBadge,
+  EuiSpacer,
+  EuiSplitPanel,
+  EuiText,
+} from '@elastic/eui';
+import { asyncForEach } from '@kbn/std';
+import { ViewMode } from '@kbn/embeddable-plugin/public';
+import type { Module } from '../../../../common/types/modules';
+import { useDashboardService } from '../../services/dashboard_service';
+import { LABELS, type LabelId } from './overview_tab_content';
+import type { KibanaAssetType } from './flyout';
+
+interface Props {
+  module: Module;
+  selectedKibanaSubTab?: KibanaAssetType;
+}
+
+export const KibanaTabContent: FC<Props> = ({ module, selectedKibanaSubTab }) => {
+  const [dashboardUrls, setDashboardUrls] = useState<Record<string, string>>({});
+  const dashboardService = useDashboardService();
+
+  useEffect(
+    function setUpDashboardUrls() {
+      const dashboards = module.kibana?.dashboard ?? [];
+      const dashboardIds = dashboards.map(({ id }) => id);
+      const urls = {};
+
+      async function getDashboardUrls() {
+        const result = await dashboardService.fetchDashboardsById(dashboardIds);
+
+        await asyncForEach(result, async ({ id }) => {
+          const url = await dashboardService.getDashboardUrl(id, ViewMode.VIEW);
+          if (url) {
+            urls[id] = url;
+          }
+        });
+
+        if (Object.keys(urls).length > 0) {
+          setDashboardUrls(urls);
+        }
+      }
+      if (dashboards.length > 0) {
```

Comment:
How difficult is it to add links to Discover for any searches, and to Visualize for any visualizations?

Reply:
Added in f9a9bbc428dd0f41be050b0c3b22c63d69cea6f3